src/common/transformations/src/transformations/op_conversions/convert_mvn1_to_mvn6.cpp

```diff
@@ -31,13 +31,18 @@ ov::pass::ConvertMVN1ToMVN6::ConvertMVN1ToMVN6() {
         if (input_rank.get_length() <= start_axis) {
             return false;
         }
+
+        const auto eps = mvn_node->get_eps();
+        if (eps > std::numeric_limits<float>::max() || eps < std::numeric_limits<float>::min())
```

Comment:
Is there a need for the `eps` value compatibility, if the `normalize_variance` attribute is set `false`?

Reply:
It doesn't affect the transformation test - it doesn't compare values, just the graph. However I'll change it accordingly to avoid confusion.